<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editcomputer</class>
 <widget class="QDialog" name="editcomputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_enterName">
     <property name="text">
      <string>Enter Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Name">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>This is a text input, you can use it to enter the name of the computer.</string>
     </property>
     <property name="placeholderText">
      <string>Enter the name of the new Computer.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_enterYear">
     <property name="text">
      <string>Enter Year of Creation:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDateEdit" name="dateEdit_year">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>This is a year selection, you can use it to enter the year the computer was built.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1752</year>
       <month>9</month>
       <day>14</day>
      </datetime>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::YearSection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_chooseType">
     <property name="text">
      <string>Select Computer Type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBox_type">
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>This is a selection input, you can use it to enter the type of the computer.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_chooseBuilt">
     <property name="text">
      <string>Was this Computer built?</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="checkBox_built">
     <property name="accessibleName">
      <string/>
     </property>
     <property name="text">
      <string notr="true">Built</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_enterDescription">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit_insertDescription">
     <property name="whatsThis">
      <string>This is a text input, you can use it to enter the description of the computer.</string>
     </property>
     <property name="placeholderText">
      <string>Enter a detailed description of the computer.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox_editComputerFinished">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
